cypress/component/SignIn/sign_in_button.spec.js

```diff
@@ -42,14 +58,15 @@ describe('Sign In: Component Loads', function () {
   it('Sign In: On Success', function () {
     cy.viewport(600, 300);
     cy.stub(Auth, 'signIn').returns(Promise.resolve(mockOidcUser));
-    cy.stub(User, 'getMe').returns(duosUser);
+    cy.intercept({method: 'GET', url: '**/api/user/me'}, {statusCode: 200, body: duosUser}).as('getMe');
```

Comment:
can you explain why we need to move from stubbing these calls to intercepting them?

Reply:
Previously, we were calling `then` on the `click` event. This would take advantage of the stubbing. That pattern is discouraged by Cypress (i.e., it is unsafe to chain calls off of click: https://docs.cypress.io/api/commands/click). So, to get around that unsafeness, instead we use the intercept + wait pattern seen on line 64 here. To do that, we can't use a stub, we have to wait for the result of the `getMe` call, and then check for the expectations we want to see.